testsuite/features/support/cobbler_test.rb

```diff
@@ -94,20 +104,20 @@ def distro_create(name, kernel, initrd, breed = 'suse')
   def profile_create(name, distro, location)
     begin
       profile_id = @server.call('new_profile', @token)
-    rescue
-      raise 'creating profile failed.' + $ERROR_INFO.to_s
+    rescue StandardError
+      raise(StandardError, "Creating profile failed. #{$ERROR_INFO}")
     end
     begin
       @server.call('modify_profile', profile_id, 'name', name, @token)
       @server.call('modify_profile', profile_id, 'distro', distro, @token)
       @server.call('modify_profile', profile_id, 'kickstart', location, @token)
-    rescue
-      raise 'modifying profile failed.' + $ERROR_INFO.to_s
+    rescue StandardError
+      raise(StandardError, "Modify profile failed. #{$ERROR_INFO}")
```

Comment:
```suggestion
      raise(StandardError, "Modifying profile failed. #{$ERROR_INFO}")
```
I know, I know, greatest nitpick ever